.github/workflows/pip-release.yml

```diff
@@ -1,46 +1,125 @@
-name: Pypi Release
-
-permissions:
-  contents: write
+# This file is autogenerated by maturin v0.14.17
+# To update, run
+#
+#    maturin generate-ci github --zig
+#
```

Comment:
Is this still true? You pushed some manual edits to this file. I assume those would be overwritten by regenerating the file?

Reply:
Yeah agreed. I'll correct it.